### ISSUE TITLE
Securing a Desktop/Laptop computer

### ISSUE THREAD

**Issue by ReitzBytes** [JonahAragon/guides.privacytools.io#3]
Varying levels of security (Higher is better)

1.  Windows 10
    - [ ] Disabling Telemetry
    - [ ] Home vs Pro (Pro is more secure, has real Bitlocker, etc)
    - [ ] Enabling BitLocker
2.  ~~Older versions of Windows (?)~~
3.  MacOS
    - [ ] Firewall applications(?) Little Snitch, etc.
    - [ ] Enabling FileVault
4.  Ubuntu-based distribution of Linux
5.  Debian/Fedora based distributions
6.  PureOS/Trisquel
7.  QubesOS
    - [ ] Hardware Requirements, Installation
    - [ ] Qubes: How containers work

**Comment by jonaharagon** [comment]
>Older versions of Windows (?)

Yeah just a big "Older versions of Windows are not secure, and therefore don't faciliate privacy" warning lol

What are you thinking here? Just a list of secure operating systems ranked from best to worst or guides on how to secure all of these options?

**Comment by ReitzBytes** [comment]
My thought was mainly for windows 7, which has security updates until January 14, 2020 and doesn't include all the telemetry in windows 10.

Definitely guides for how to secure all these - or links to how to install them. Just something so people have something a little more user-friendly than the landing pages of the recommended distributions.

**Comment by jonaharagon** [comment]
> Ubuntu-based distribution of Linux
> Debian/Fedora based distributions

This really seems more down to personal preference than a security issue. But we could come up with some recommended distros from the mix. Elementary OS for total beginners all the way up to maybe Manjaro/Arch for more advanced use-cases. But emphasize even something like Manjaro is fairly beginner-friendly nowadays.

> My thought was mainly for windows 7, which has security updates until January 14, 2020

That's not a lot of time and I have a hard time recommending older systems that will shortly be out of date. We should focus on Windows and maybe emphasize alternatives to W10 that aren't Windows based.

I edited your comment with a checklist because I want these issues to have a to-do list when possible we can keep up to date, feel free to add any specific topics under each category.

**Comment by ReitzBytes** [comment]
> > Ubuntu-based distribution of Linux
> > Debian/Fedora based distributions
> 
> This really seems more down to personal preference than a security issue. But we could come up with some recommended distros from the mix. Elementary OS for total beginners all the way up to maybe Manjaro/Arch for more advanced use-cases. But emphasize even something like Manjaro is fairly beginner-friendly nowadays.

> That's not a lot of time and I have a hard time recommending older systems that will shortly be out of date. We should focus on Windows and maybe emphasize alternatives to W10 that aren't Windows based.

Agreed! 

> I edited your comment with a checklist because I want these issues to have a to-do list when possible we can keep up to date, feel free to add any specific topics under each category.

Perfect. Looks good!
